Makefile

```diff
@@ -1,5 +1,6 @@
 SHELL:=/bin/bash
-L0_VERSION?=$(shell git describe --tags)
+L0_VERSION:=$(shell git describe --tags | sed "s/v//")
```

Comment:
This would break our versioning, this must be reverted back. Remember, we run this all the time, not just on "clean" releases like v0.10.4, but also on working branches where the version may be some long hash.